HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,141 @@
-
-
-### Graph 4 
-```{r, echo = T}
-data(mpg)
-
-#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
-library(RColorBrewer)
-display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
-```
-
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
-
-
-
-Now using the above mpg dataset, make this graph
-
-```{r graph 4 code}
-
-```
-
-
-
-
+---
+title: "HW02_B_Graph-Mimic"
+author: "YOUR NAME HERE"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+ggplot(diamonds, aes(cut, color = clarity, fill = clarity)) + #plot cut, color/fill based on clarity
+  geom_bar(position = "dodge") + #write bars, dodged
+  labs(x = "Diamond Cut", y = "Number of Diamonds", title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by\ntype of cut quality and clarity of diamond") + #add all labels
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") + #add caption
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.4) #add highlight rect
+  
+
+
+
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+iris$SpeciesOrdered <- factor(iris$Species, levels = c("versicolor", "setosa","virginica")) #good order
+
+iris_plot <- ggplot(iris, aes(Sepal.Length, Petal.Length,color = Species, shape = Species))
+
+iris_plot + facet_wrap(vars(SpeciesOrdered),scales = "free") + #make three subplots
+  scale_color_manual(values = c("green","red","dodgerblue2")) + #specify colors
+  scale_shape_manual(values = c(17, 16, 15)) + #specify shapes
```

Comment:
Good on you for specifying colors and shapes this way, I just assigned both shape and color to species in my original ggplot line and it wound up churning out the right ones, but this might not have been the case for you since you reordered the graphs how we were supposed to.